scripts/linux/runE2ETest.sh

```diff
@@ -15,6 +15,7 @@ function clean_up() {
 
     echo 'Remove IoT Edge and config file'
     rm -rf /var/lib/aziot/edged/
+    rm -rf /etc/systemd/system/aziot-edged.service.d/
```

Comment:
We will likely need this in trcE2ETest.sh as well.

Reply:
Added